common/ethnicities/wc_ethnicities_elven.txt

```diff
@@ -973,4 +973,7 @@ naga_ethnicity = {
 	gene_grade = {
```

Comment:
its the issue you described before, because the webbing is a 2d mesh. I will try duplicating it so it has front faces on both sides and see if it helps.

Reply:
fixed a previous issue where a mesh had some quads causing lighting problems, + fixed some UV issues. Game does not like double vertexes tho in some instances
![image](https://user-images.githubusercontent.com/46745564/126917503-b764aa80-336d-45b7-a1eb-ca23c0fb09b7.png)

Reply:
![image](https://user-images.githubusercontent.com/46745564/126917783-94ff354d-d562-445d-a513-7e1a263cfe3d.png)
all backfaces are really shiny